### PR TITLE
Removes the Clarke breaking floor tiles on move

### PR DESCRIPTION
**Pull request by TheVekter**
## About The Pull Request

Exactly what it says on the tin. If y'all think it will make the Clarke unbalanced somehow I'm down to nerf it in some other way if someone wants to make a suggestion.

## Why It's Good For The Game

We've had too many people build these for the sake of just tooling around the station annoying people by breaking floor tiles. Aside from it having tracks it doesn't make too much sense to break tiles anyway.

## Changelog
:cl:
tweak: The Clarke's weight has been reduced, causing it not to break floor tiles anymore.
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
I'm sorry! I just wanted to drift in my mech!

**Comment by TheVekter** [comment]
I'm more referring to the two posibrains last round on Manuel doing it, but yeah

**Comment by SumFaggotPlayTester** [comment]
Yeah but m u l t i   t r a c k  d r i f t i n g  in a mech is worth it.

**Comment by HomocidlePancakes** [comment]
>coder halp botany is using blood tomatoes again!1!1!1

Like, I understand that this just fluff, and we won’t loose anything by removing it, but your reasoning is just stupid. If someone is doing something annoying, robust them, don’t go to git.

**Comment by TheVekter** [comment]
> 
> 
> > coder halp botany is using blood tomatoes again!1!1!1
> 
> Like, I understand that this just fluff, and we won’t loose anything by removing it, but your reasoning is just stupid. If someone is doing something annoying, robust them, don’t go to git.

The difference here is that the Clarke is _kinda_ hard to kill if the person using it knows what they're doing. I would normally agree, but there's no real reason this should exist aside from flavor and it's annoying enough that I'd rather remove it.

**Comment by Tlaltecuhtli** [comment]
its as hard as sticking the mech beacon on it and law 2 the ai to emp spam it

**Comment by FloranOtten** [comment]
What use are floortiles, anyhow. What's the bad part about them breaking beyond it looking a bit bad?

**Comment by ATH1909** [comment]
For what it's worth, I've seen people on fulp complain about this as well, even when people weren't actively trying to break most of the floor tiles on the station.

For the record, I'm neutral towards this PR.

**Comment by HomocidlePancakes** [comment]
If they went an extra step of making a mech just so they can do something a crowbar can, then I don’t see an issue with them being sturdier.

**Comment by RaveRadbury** [comment]
> If they went an extra step of making a mech just so they can do something a crowbar can, then I don’t see an issue with them being sturdier.

This is not an accurate comparison.

**Comment by necromanceranne** [comment]
Not even a crowbar can match the destructive, and I mean actually destructive, power of the clarke. It's not just ripping up the tile. It's destroying the hull.

**Comment by HomocidlePancakes** [comment]
Why the fuck can it cause breaches by just moving? What the fuck. Gut it.

File: code/game/mecha/working/clarke.dm [hunk 1]
```diff
@@ -48,24 +48,6 @@
...
-	if (.)
-		if(prob(30))
-			crush_floor()
-
-/obj/mecha/working/clarke/mechturn(direction)
-	if(prob(80))
-		crush_floor()
-	return ..()
-
-///Damages floor tiles and plating as the Clarke moves over them with its tracks. Damage is superficial, will not create hull breaches.
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/mecha/working/clarke.dm:62]: this would suggest it is not intentional that this proc is suppose to create hull breaches.

Comment (TheVekter) [reply code/game/mecha/working/clarke.dm:62]: I haven't seen it cause breaches, but it does break plating, which is more annoying to replace.

**Comment by TheVekter** [comment]
Anti-stale.

**Pull request merged**